{{ cookiecutter.package_name }}/setup.py

```diff
@@ -33,13 +35,18 @@ def load_about():
         "Issue tracker": "{{ cookiecutter.git_repo }}/issues",
     },
     license="{{ cookiecutter.license }}",
-    author="{{ cookiecutter.author }}",
-    description="{{ cookiecutter.plugin_name}} plugin for Tutor",
+    description="{{ cookiecutter.description }}",
     long_description=load_readme(),
+    long_description_content_type="text/x-rst",
```

Comment:
:+1: This is very useful, as it will remove a warning when uploading the plugin to pypi.